drivers/base/bootvid/common.c

```diff
@@ -1,3 +1,13 @@
+/*
+ * PROJECT:     ReactOS Boot Video Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Platform-independent common helpers and defines
+ * COPYRIGHT:   Copyright 2010 Gregor Schneider <Gregor.Schneider@reactos.org>
```

Comment:
```suggestion
 * COPYRIGHT:   Copyright 2010 Gregor Schneider <gregor.schneider@reactos.org>
```
(email addresses are case insensitive, so prefer lowercase)

Reply:
yep. But I just took what's already in our tree. I'll change it though.